index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Calculadora</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>𝙏𝙖𝙗𝙪𝙖𝙙𝙖</h1>
+      <hr />
+    </header>
+    <main>
+      <form id="form">
+      <label id="inputMULT">
+        <div>
+            <span id="inputText">Multiplicador</span>
+        </div>
+        <input id="input" type="number" />
+        <button id="button"> Calcular </button>
```

Comment:
Este não é um simples `button`, é um button de submição de formulário por isso tem de ter a propriedade `type="submit"`
```suggestion
        <button type="submit" id="button">Calcular</button>
```

https://www.w3schools.com/tags/tag_button.asp